Alignment/OfflineValidation/test/PVValidation_T_cfg.py

```diff
@@ -0,0 +1,311 @@
+process.filterOutLowPt.runControlNumber = int(runboundary)
+
+if isMC:
+     process.goodvertexSkim = cms.Sequence(process.noscraping+process.filterOutLowPt)
+else:
+     process.goodvertexSkim = cms.Sequence(process.primaryVertexFilter + process.noscraping + process.filterOutLowPt)
+
+####################################################################
+# Load and Configure Measurement Tracker Event
+# (this would be needed in case NavigationSchool is set != from ''
+####################################################################
+# process.load("RecoTracker.MeasurementDet.MeasurementTrackerEventProducer_cfi") 
+# process.MeasurementTrackerEvent.pixelClusterProducer = 'TRACKTYPETEMPLATE'
+# process.MeasurementTrackerEvent.stripClusterProducer = 'TRACKTYPETEMPLATE'
+# process.MeasurementTrackerEvent.inactivePixelDetectorLabels = cms.VInputTag()
+# process.MeasurementTrackerEvent.inactiveStripDetectorLabels = cms.VInputTag()
+
+####################################################################
+# Load and Configure TrackRefitter
+####################################################################
+# process.load("RecoTracker.TrackProducer.TrackRefitters_cff")
+# import RecoTracker.TrackProducer.TrackRefitters_cff
+# process.TrackRefitter = RecoTracker.TrackProducer.TrackRefitter_cfi.TrackRefitter.clone()
+# process.TrackRefitter.src = "TRACKTYPETEMPLATE"
+# process.TrackRefitter.TrajectoryInEvent = True
+# process.TrackRefitter.NavigationSchool = ''
+# process.TrackRefitter.TTRHBuilder = "WithAngleAndTemplate"
+
+####################################################################
+# Load and Configure common selection sequence
+####################################################################
+import Alignment.CommonAlignment.tools.trackselectionRefitting as trackselRefit
+process.seqTrackselRefit = trackselRefit.getSequence(process,'TRACKTYPETEMPLATE')
+process.HighPurityTrackSelector.trackQualities = cms.vstring()
+process.AlignmentTrackSelector.ptMin     = cms.double(0.)
+process.AlignmentTrackSelector.nHitMin2D = cms.uint32(0)
+process.AlignmentTrackSelector.nHitMin   = cms.double(0.)
+process.AlignmentTrackSelector.d0Min     = cms.double(-999999.0)
+process.AlignmentTrackSelector.d0Max     = cms.double(+999999.0)                  
+process.AlignmentTrackSelector.dzMin     = cms.double(-999999.0)
+process.AlignmentTrackSelector.dzMax     = cms.double(+999999.0)  
+
+####################################################################
+# Debugging facility
+####################################################################
+#print process.seqTrackselRefit
+#print process.seqTrackselRefit.dumpSequencePython
+
+#obj = process.AlignmentTrackSelector
+#obj = process.HighPurityTrackSelector
+
+#for i in obj._Parameterizable__parameterNames:
+#     print i 
+
+#itemDir = obj.__dict__
+#for i in itemDir:
+#    print '{0}  :  {1}'.format(i, itemDir[i])
+
+#for attr in dir(obj):
+#    print "obj.%s = %s" % (attr, getattr(obj, attr))
```

Comment:
@mmusich do you still need this debugging faciltity or do you trust the tool enough now?